kolibri/plugins/management/assets/src/actions.js

```diff
@@ -25,19 +25,30 @@ const samePageCheckGenerator = require('kolibri.coreVue.vuex.actions').samePageC
  * the API to state in the Vuex store
  */
 
-function _userState(data) {
-  // assume just one role for now
+function _stateUser(apiUserData) {
+  // handle role representation
   let kind = UserKinds.LEARNER;
-  if (data.roles.length && data.roles[0].kind === 'admin') {
-    kind = UserKinds.ADMIN;
-  }
+
+  // makes kind the highest ranking role
+  apiUserData.roles.forEach(role => {
+    // using a switch statement. Checks all in order of heirarchy
+    switch(role.kind){
+      // sets role to admin 
+      case UserKinds.ADMIN || UserKinds.SUPERUSER:
+        kind = UserKinds.ADMIN;
+        break;
+      case UserKinds.COACH:
+        if(kind != UserKinds.ADMIN) kind = UserKinds.COACH;
```

Comment:
with the conditional and expression all on one line this might be okay, but in general it's a bit safer and more explicit to always include brackets: 

http://stackoverflow.com/questions/4797286/are-curly-braces-necessary-in-one-line-statements-in-javascript